lib/fluent/plugin/out_cloudwatch_logs.rb

```diff
@@ -309,6 +321,7 @@ def find_log_stream(group_name, stream_name)
           break
         end
         next_token = response.next_token
+        sleep 0.1
```

Comment:
To reduce the number of describe_log_streams calls. (avoid rate exceeded error).
But It may be better to use aws-skd exponential backoff.